README.md

```diff
@@ -1 +1,31 @@
-# react-todo-app
\ No newline at end of file
+## React Tutorial: A Comprehensive Guide for Beginners
+
+This repository holds the code for a React todos application. You can follow the step-by-step [guide to building the fantastic project here](https://ibaslogic.com/react-tutorial-for-beginners/).
+
+## [Click to See the Todos App](https://react-todos-ibas.vercel.app/)
```

Comment:
I noticed the live demo here leads to another project. In future you could consider deploying your project to Github Pages.

Reply:
Thanks